code/modules/reagents/chemistry/reagents/atmos_gas_reagents.dm

```diff
@@ -128,8 +131,10 @@
 			continue
 
 		organ_being_healed.apply_organ_damage(-0.5 * REM * seconds_per_tick, required_organ_flag = ORGAN_ORGANIC)
+		. = TRUE
 
-	return ..()
+	..()
+	return .
```

Comment:
```
	. = ..()
	for(var/obj/item/organ/organ_being_healed as anything in breather.organs)
		if(!organ_being_healed.damage)
			continue

		organ_being_healed.apply_organ_damage(-0.5 * REM * seconds_per_tick, required_organ_flag = ORGAN_ORGANIC)
		. = TRUE
```
Doesn't seem to rely on `current_cycle` or reagent volume so moving the super call shouldn't change the logic.